db/migrations/20190913165643-add-indexes-to-unfurls-table.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  up: (queryInterface) => Promise.all([
+    queryInterface.addIndex('Unfurls', {
+      fields: ['channelSlackId', 'slackWorkspaceId', 'slackUserId', 'githubRepoId'],
+      name: 'unfurls_channel_slack_id_slack_workspace_id_slack_user_id_github_repo_id',
```

Comment:
These names are a bit long. Postgres has an identifier limit of 64, so the max # of characters is 63.
This is 72 characters, and will be truncated.

https://www.postgresql.org/docs/9.1/sql-syntax-lexical.html

This might work:

```suggestion
      name: 'unfurls_slack_channel_workspace_user_ids_github_repo_id_idx',
```